custom/ewsghana/tasks.py

```diff
@@ -189,6 +190,13 @@ def migrate_email_settings(domain):
     endpoint = GhanaEndpoint.from_config(config)
     migrate_email = EmailSettingsSync(domain)
 
+    for user in iter_docs(WebUser.get_db(), WebUser.ids_by_domain(domain)):
+        for notification in ReportNotification.by_domain_and_owner(domain, user['_id'], stale=False):
+            notification.delete()
```

Comment:
Ok, I don't think deleting and resyncing is an option either because people could have created report notifications in the meantime that we don't want to delete.

What's the issue that this pr is trying to solve, just to stop sending the emailed reports for reports that use an archived location as the filter?